libr/core/cmd_write.c

```diff
@@ -1788,3 +1843,66 @@ static int cmd_write(void *data, const char *input) {
 	R_FREE (ostr);
 	return 0;
 }
+
+static void cmd_write_init(RCore *core, RCmdDesc *parent) {
+	DEFINE_CMD_DESCRIPTOR (core, w);
+	DEFINE_CMD_DESCRIPTOR (core, wa);
+	DEFINE_CMD_DESCRIPTOR (core, wA);
+	DEFINE_CMD_DESCRIPTOR (core, wc);
+	DEFINE_CMD_DESCRIPTOR (core, we);
+	DEFINE_CMD_DESCRIPTOR (core, wo);
+	DEFINE_CMD_DESCRIPTOR (core, wop);
+	DEFINE_CMD_DESCRIPTOR (core, wp);
+	DEFINE_CMD_DESCRIPTOR (core, wt);
+	DEFINE_CMD_DESCRIPTOR (core, wv);
+	DEFINE_CMD_DESCRIPTOR (core, wx);
+
+	RCmdDesc *wv_cd = r_cmd_desc_argv_new (core->rcmd, parent, "wv", wv_handler);
+	r_return_if_fail (wv_cd);
+	RCmdDescHelp wv_help;
+	r_cmd_desc_help_init (&wv_help);
+	wv_help.usage = "wv[size] [value]";
+	wv_help.summary = "Write value as 4-bytes/8-bytes based on value";
+	wv_help.args_str = "[value]";
+	wv_help.description = "Write the number passed as argument at the current offset as a 4-bytes value or 8-bytes value if the input is bigger than UT32_MAX, respecting the cfg.bigendian variable";
+	wv_help.group_summary = "Write value of given size";
+	RCmdDescHelp wv_ex1 = { "wv 0xdeadbeef", "Write the value 0xdeadbeef at current offset" };
+	r_vector_push (&wv_help.examples, &wv_ex1);
+	r_cmd_desc_set_help (wv_cd, &wv_help);
+
+	RCmdDesc *wv1_cd = r_cmd_desc_argv_new (core->rcmd, wv_cd, "wv1", wv1_handler);
```

Comment:
Maybe adding a small title in a comment like `// "wv1" - bla-bla` would help visually separate these?

Reply:
Also can be compactified with a macro, I think.

Reply:
Probably this can be "macrofied" a little more, but there are some problems with the addition of examples I could not figure out right now. However, this looks already much better this way, what do you think?

By the way, by doing this we can put all the define (https://github.com/radareorg/radare2/pull/16782/files#diff-8a598c2e45468fc2d7a1e00496426b29R1847) for all commands in a single (or multiple) `.h` files, so if someone wants to change the description/summary/etc. of a command he doesn't need to mess with the code.

Reply:
How about using a format more similar to the existing help description arrays?

Reply:
I can do it!